docs/tremor-query/operators.md

```diff
@@ -137,12 +137,19 @@ This operator is deprecated please use `qos::backpressure` instead.
 
 The backpressure operator is used to introduce delays based on downstream systems load. Longer backpressure steps are introduced every time the latency of a downstream system reached `timeout`, or an error occurs. On a successful transmission within the timeout limit, the delay is reset.
 
+The operator supports two modes:
+
+- `discard` - the standard, when backpressure is triggered it will discard new messages. This is designed to fulfill the need of low transport latency at the cost of loss.
```

Comment:
```suggestion
- `discard` - the standard, when backpressure is triggered it will discard new messages by sending them to the `overflow` output port. This is designed to fulfill the need of low transport latency at the cost of loss.
```